masterlist.yaml

```
@@ -21155,6 +21155,21 @@ plugins:
         itm: 6
         udr: 1
 
+  - name: 'Street Vendors of Cyrodiil.esp'
+    url:
+      - link: 'https://www.nexusmods.com/oblivion/mods/48143/'
+        name: 'Street Vendors of Cyrodiil'
+    inc: [ 'Better Cities Resources.esm' ]
```

You could use the `BetterCities` file anchor here:
```suggestion
    inc: [ *BetterCities ]
```